pkg/apis/kops/componentconfig.go

```diff
@@ -1055,6 +1055,12 @@ type AWSLoadBalancerControllerConfig struct {
 	Enabled *bool `json:"enabled,omitempty"`
 	// Version is the container image tag used.
 	Version *string `json:"version,omitempty"`
+	// EnableUseOfWAFClassic specifies whether the controller can use WAFs (Classic Regional).
+	// Default: false
+	EnableUseOfWAFClassic *bool `json:"enableWAFClassic,omitempty"`
+	// EnableUseOfWAF specifies whether the controller can use WAFs (V2).
+	// Default: false
+	EnableUseOfWAF *bool `json:"enableWAF,omitempty"`
```

Comment:
I think we can drop the `UseOf` part.
Any special reason to use pointers?
```suggestion
	// EnableWAFClassic specifies whether the controller can use WAFs (Classic Regional).
	// Default: false
	EnableWAFClassic bool `json:"enableWAFClassic,omitempty"`
	// EnableWAF specifies whether the controller can use WAFs (V2).
	// Default: false
	EnableWAF bool `json:"enableWAF,omitempty"`
```

Reply:
I was following the example from #12074. I can use a `bool` directly. That would cut out the need for `fi.BoolValue` elsewhere.

Reply:
`Enabled` is special use case. `bool` should be enough when default is `false`.